### PR TITLE
Update examples to reflect shortcuts

### PR DESCRIPTION
**Pull request by cdrage**
Updates all the examples to reflect the "kedge" keys that enable
reducing the Kubernetes artifact file length. Such as portMappings and
health.

### CODE REVIEW

File: examples/gitlab/gitlab.yml [hunk 1]
```diff
@@ -1,18 +1,33 @@
...
+
 containers:
 - name: gitlab
   image: gitlab/gitlab-ce:9.4.1-ce.0
   imagePullPolicy: ""
+  health:
+    httpGet:
+      path: /help
+      port: 80
+    initialDelaySeconds: 200
```
Comment (kadel) [inline examples/gitlab/gitlab.yml]: readiness and liveliness probes originally had different `initalDelaySeconds`.

But I suppose that for purpose of the demo it is  ok to consolidate (maybe)
What @surajnarwade thinks about this ?

**Comment by surajssd** [comment]
We did not want to have the shortcuts for health, because they were taken from the upstream so we might want to consider before making those changes. We will only use shortcuts like `health` when all the values are same!

**Comment by cdrage** [comment]
@surajssd Then what's the point of `health` when both readiness and liveness probe's often have different values? :-1: Sure it's a good shortcut and cuts down the file-length, but I've rarely seen both readiness and liveness probes being the same values. Maybe this is good for a different discussion!

**Comment by surajssd** [comment]
@cdrage in most cases it is same, but not in this particular case!

**Comment by cdrage** [comment]
@surajssd More.. both examples are different here. I guess I'll change it back!

**Comment by cdrage** [comment]
Examples fixed, HOWEVER. We are blocked on #438 . Got to wait until #438 is fixed before this will work.

**Comment by cdrage** [comment]
Re-based after port mappings has been merged, integration + unit tests pass.

Ready for review :100:

**Comment by cdrage** [comment]
Since we've released #486 I'm going to go ahead with the examples merge! :+1:

**Pull request merged**